user-guide/Advanced_Modules/DOM/DOM_best_practices.md

```diff
@@ -0,0 +1,200 @@
+
+**Not recommended:**
+
+<div class="mermaid">
+classDiagram
+    direction RL
+    `Incident Info` <-- `Incidents`
+    `Performed Actions` <-- `Incidents` : Multiple sections
+    class `Incidents`{
+        &lt;&lt;DomDefinition&gt;&gt;
+    }
+    class `Incident Info`{
+        &lt;&lt;SectionDefinition&gt;&gt;
+        Name
+        Description
+        Creator
+        Element IDs
+    }
+    class `Performed Actions`{
+        &lt;&lt;SectionDefinition&gt;&gt;
+        Technician
+        Action
+        Notes
+    }
+</div>
+
+**Recommended:**
+
+<div class="mermaid">
+classDiagram
+    direction RL
+    `Incident Info` <-- `Incidents`
+    `Performed Actions` <-- `Incidents Metadata` : Multiple sections
+    `Incident Info` --> `Incidents Metadata`
+    class `Incidents`{
+        &lt;&lt;DomDefinition&gt;&gt;
+    }
+    class `Incidents Metadata`{
+        &lt;&lt;DomDefinition&gt;&gt;
+    }
+    class `Incident Info`{
+        &lt;&lt;SectionDefinition&gt;&gt;
+        Name
+        Description
+        Creator
+        Element IDs
+        Metadata
+    }
+    class `Performed Actions`{
+        &lt;&lt;SectionDefinition&gt;&gt;
+        Technician
+        Action
+        Notes
+    }
+</div>
+
+### Use the most efficient FieldDescriptor
+
+The way data is stored can impact the size of a `DomInstance`. Preferably, the data should convey the same meaning using the least amount of bytes. An example of this is a 'severity' field, which could contain "Critical", "Minor" or "Major". Instead of defining these as string values every time, it is advisable to use a `GenericEnumFieldDescriptor<int>` instead. Each severity is then mapped to a number, and only that number is stored in the `DomInstance`. This can reduce the required bytes when compared to storing long string values.
+
+### Avoid reusing SectionDefinitions if not all fields are applicable
+
+With `SectionDefinitions`, it is possible to reuse them inside a DOM manager. If all fields inside a `SectionDefinition` are applicable to both `DomDefinitions`, it may be advisable to do so. However, if only a subset is applicable, it is better to define a new `SectionDefinition` specific to the other `DomDefinition`. This ensures that only relevant data is stored.
+
+### Try to limit the amount of statuses and fields allowed per state
+
+On the `DomBehaviorDefinition`, you have very fine control over what fields should be filled in every state. However, this can result in a large configuration when many fields and statuses are used. To reduce the size of the `DomBehaviorDefinition`, it is recommended to limit the number of fields used in a status system. As mentioned in an earlier recommendation, it could be beneficial to split off data not tied to a specific state into a separate `DomInstance`.
```

Comment:
This last sentence is not quite clear to me, do you mean creating another DomDefinition to separate non-state specific data? In that case, I think the following would be more accurate:
```suggestion
On the `DomBehaviorDefinition`, you have very fine control over what fields should be filled in every state. However, this can result in a large configuration when many fields and statuses are used. To reduce the size of the `DomBehaviorDefinition`, it is recommended to limit the number of fields used in a status system. As mentioned in an earlier recommendation, it could be beneficial to split off data not tied to a specific state into a separate `DomDefinition`.
```